volatility3/framework/plugins/windows/handles.py

```diff
@@ -57,10 +57,11 @@ def _decode_pointer(self, value, magic):
 
```

Comment:
It looks like this function does not do anything similar to the `_decode_pointer` function in handles.
It just essentially removes the 2 lowest bits. Since `_PORT_ASSIGNMENT_ENTRY`'s `Entry`  is of type pointer and not a signed long value there is no real need to add the `get_canonical_address`. Of course adding it will work but nothing will change.